index.html

```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Koala Kounter</title>
-  <script src="script.js"></script>
-  <!-- <link rel="stylesheet" type="text/css" href="styles.css"> -->
-</head>
-<body>
-  <img src="https://1.bp.blogspot.com/-KZdHaJHLaTQ/TmPwhzbEekI/AAAAAAAAE3U/Y6PnMbo6kSc/s1600/Funny+Koala2.jpg" id="pic" alt="click the koala">
+	<head>
+		<title id="title">Koala Kounter</title>
+		<script src="script.js"></script>
+		<!-- <link rel="stylesheet" type="text/css" href="styles.css"> -->
+	</head>
+	<style>
+		.color {
+			color: red;
+			font-size: 2rem;
+		}
+	</style>
+	<body>
+		<img src="https://1.bp.blogspot.com/-KZdHaJHLaTQ/TmPwhzbEekI/AAAAAAAAE3U/Y6PnMbo6kSc/s1600/Funny+Koala2.jpg" id="pic" alt="click the koala">
```

Comment:
Indent with spaces, not tabs! Github reads tabs as 8 spaces so it looks wacky like this, hard to read.